Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -436,6 +436,54 @@ Next, navigate to a new website in Safari. You should see traced function calls
  21324 ms     | -[NSURLRequest initWithURL:0x106388b00 cachePolicy:0x0 timeoutInterval:0x106388b80
 ```
 
+### Binary Analysis
+
+An introduction to binary analysis using binary analysis framework has already been discussed in [Dynamic Analysis](0x05c-reverse-engineering-and-tampering#dynamic-analysis "Dynamic analysis") section for Android. It is recommended before moving forward, to revisit and refresh the concepts. 
```

Comment:
Let's try to use an active voice as much as we can:
```suggestion
An introduction to binary analysis using binary analysis framework has already been discussed in [Dynamic Analysis](0x05c-reverse-engineering-and-tampering#dynamic-analysis "Dynamic analysis") section for Android. We recommend you to revisit this section and refresh the concepts on this subject.
```

Reply:
My bad, trying to minimize these mistakes.